docs/manual/How to contribute to the development of the tool.rst

```diff
@@ -0,0 +1,27 @@
+=======================================================
+How to contribute to the development of the tool
+=======================================================
+
+All the community is encouraged to collaborate to the maintenance and development of OSeMOSYS by proposing code modifications, enhancements and extensions. The proposers may want to have the modifications reviewed by experts and included in a bi-annual peer-reviewed publication on the status of development of OSeMOSYS. In such case, they shall submit a related `OSeMOSYS Enhancement Proposal (OEP) <https://github.com/OSeMOSYS/OSeMOSYS/blob/master/.github/ISSUE_TEMPLATE/osemosys-enhancement-proposal.md>`_ on the `OSeMOSYS GitHub Repository <https://github.com/OSeMOSYS/OSeMOSYS>`_ following the instructions below:

```

Comment:
Link to the issue tracker `https://github.com/OSeMOSYS/OSeMOSYS/issues/new?assignees=abhishek0208%2C+tniet%2C+willu47&labels=OEP&template=osemosys-enhancement-proposal.md&title=%5BOEP%5D+%3CReplace_with_name_of_proposal%3E`